prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    // Your code here
+    document.addEventListener('DOMContentLoaded', contentLoaded);
+    function contentLoaded() {
+      console.log('content loaded');
+    }
+
+    window.addEventListener('mousemove', mouseMove);
+    function mouseMove() {
+      console.log('mouse is moving!');
+    }
+
+    var $helloWorld = document.querySelector('button.hello-world-btn');
+    addEventListener('click', hello);
```

Comment:
Be sure to add the event listener directly to the button so that only clicking the button logs hello world.

```js
$hellowWorld.addEventListener('click', hello);
```

Reply:
![image](https://user-images.githubusercontent.com/90783131/150875861-ba08e23e-9f4e-4026-ba52-25a7e70b9cec.png)